src/river/layers/water_rasterized.py

```
@@ -0,0 +1,103 @@
+import processing
+from qgis.core import QgsVectorLayer, QgsCoordinateReferenceSystem
```

Ruff (0.11.9)

`qgis.core.QgsVectorLayer` imported but unused

Remove unused import: `qgis.core.QgsVectorLayer`

(F401)